omod/src/main/resources/config.xml

```diff
@@ -516,7 +516,7 @@
 		<property>eptsreports.treatmentFollowUpUuid</property>
 		<defaultValue>93603742-1cae-4970-9077-e2b27e46bd7e</defaultValue>
 		<description>
-			Seguimento de tratamento TPT
+			UUID for the Seguimento de tratamento TPT concept
```

Comment:
![image](https://user-images.githubusercontent.com/7911110/132228492-8f3f3b08-29fc-4afd-ab7b-f2aab1665d99.png)
I see this blank field what is it about?